### PR TITLE
Put "Configuring your app" under "Configuring Ember"

### PR DESCRIPTION
**Pull request by joostdevries**
This might be a good location for now.
#410

### CODE REVIEW

File: source/configuring-ember/configuring-your-app.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+Ember-CLI assigns `ENV.EmberENV` to `window.EmberENV`, which Ember reads on application initialization.
+
+Additionally, Ember-CLI contains a number of environment-dependent helpers for assets:
+
+- [Env specific assets](#environment-specific-assets)
+- [Env specific asset fingerprinting](#fingerprinting-and-cdn-urls)
+
+It is now also possible to override command line options by creating a file in your app's root directory called `.ember-cli` and placing desired overrides in it.
+
+For example, a common desire is to [change the port](http://stackoverflow.com/questions/24003944/save-port-number-for-ember-cli-in-a-config-file) that ember-cli serves the app from. It's possible to pass the port number directly to ember server in the command line, e.g. `ember server --port 8080`. If you wish to make this change a permanent configuration change, make the `.ember-cli` file and add the options you wish to pass to the server in a hash.
```
Comment (locks) [inline source/configuring-ember/configuring-your-app.md]: first sentence refers to `ember-cli` instead of `Ember-CLI`

File: source/configuring-ember/configuring-your-app.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+Ember-CLI ships with support for managing your application's environment. Ember-CLI will build an environment config file at `config/environment`. Here, you can define an ENV object for each environment (development, test and production). For now, this is limited to the three environments mentioned.
```
Comment (locks) [inline source/configuring-ember/configuring-your-app.md]: "will generate an environment config file" maybe? I associate `build` with `$ ember build`

Comment (locks) [inline source/configuring-ember/configuring-your-app.md]: I'd reword from "Here, you can …" onwards to make it clearer.

File: source/configuring-ember/configuring-your-app.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+The ENV object contains two important keys: 1) EmberENV, and 2) APP. The first can be used to define Ember feature flags (see the [Feature Flags guide](http://emberjs.com/guides/configuring-ember/feature-flags/)). The second can be used to pass flags/options to your application instance.
```
Comment (locks) [inline source/configuring-ember/configuring-your-app.md]: I think "1)" and "2)" are unnecessary since it's inline.

Comment (locks) [inline source/configuring-ember/configuring-your-app.md]: Replace "The first" and "The second" with the actual names of the keys.

File: source/configuring-ember/configuring-your-app.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+You can access these environment variables in your application code by importing from `../config/environment` or `your-application-name/config/environment`.
```
Comment (locks) [inline source/configuring-ember/configuring-your-app.md]: Like https://github.com/emberjs/guides/pull/520/files#r36064293 I think this is something we should agree on the style and stick to it, and explain it once somewhere.

File: source/configuring-ember/configuring-your-app.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+```js
+import ENV from 'your-application-name/config/environment';
+
+if (ENV.environment === 'development') {
+  // ...
+}
+```
+
+Ember-CLI assigns `ENV.EmberENV` to `window.EmberENV`, which Ember reads on application initialization.
```
Comment (locks) [inline source/configuring-ember/configuring-your-app.md]: Is this really necessary to say?

**Comment by joostdevries** [comment]
incorporated feedback + removed some redundant content + several fixes.

**Comment by michaelrkn** [comment]
On the docs call, we agreed that we really like this content, and that we'd also like to see a list of configuration options. @joostdevries is that something you can help out with?

**Comment by joostdevries** [comment]
cool with it but leaving for a week. can look into it when I get back.

**Comment by locks** [comment]
hi, I merged and opened a separate issue just to move things along. I hope I'm not out of place, see you in #568!

**Comment by michaelrkn** [comment]
:+1:
